content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -21,29 +21,109 @@ The features described on this page are in private beta.
 - **Deployment configurations**: Self-managed, RDS, RAC, Exadata
 - **Architecture**: Multi-tenant
 
-The following deployment configurations, components, and features are not supported: Oracle Autonomous Database, ASM, and Data Guard.
+The following deployment configurations, components, and features are not supported: Oracle Autonomous Database and Data Guard.
 
+## Prerequisites
 
-## Supported Agent versions
+### Install the Oracle integration
 
-The following custom build is supported: `7.44.1~dbm~oracle~beta~0.28`.
+On the [Integrations][1] page in Datadog, install the [Oracle integration][2] for your organization. This will install an [Oracle dashboard][5] in your account which can be used to monitor the performance of your Oracle databases.
 
-## Prerequisites
+### Upgrade the Oracle integration in your Agent
 
-1. **Install the Oracle integration**: On the [**Integrations**][1] page in Datadog, install the [Oracle integration][2] for your organization. This will install an Oracle dashboard in your account which can be used to monitor the performance of your Oracle databases.
-2. **Upgrade the Oracle integration in your Agent**: You can skip this step if this is your first time installing the Oracle integration. If you've previously installed the Oracle integration, migrate the legacy configuration from the `conf.d/oracle.d/` directory to the new integration path located in the `conf.d/oracle-dbm.d/` directory.
+You can skip this step if this is your first time installing the Oracle integration. If you already installed the Oracle integration, migrate the legacy configuration from the `conf.d/oracle.d/` directory to the new integration path located in the `conf.d/oracle-dbm.d/` directory.
 
 Use the following command to migrate the Oracle integration from the legacy integration to the new one:
 
-```bash
-mv /etc/datadog-agent/conf.d/oracle.d/conf.yaml /etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml
+```shell
+cp /etc/datadog-agent/conf.d/oracle.d/conf.yaml /etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml
+```
+
+Deactivate the legacy integration:
+
+```shell
+mv /etc/datadog-agent/conf.d/oracle.d/conf.yaml /etc/datadog-agent/conf.d/oracle.d/conf.yaml.bak
+```
+
+Deactivating the legacy integration prevents sending the system metrics twice.
+
+Since the Agent doesn't require an external Oracle client, remove the `jdbc_driver_path` configuration parameter from the new parameter file `/etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml`. 
+
+### Install Agent
+
+To start collecting Oracle telemetry, first install the Datadog Agent. 
+
+The Agent does not need to run on the same server nor the same platform as the monitored databases, and can be installed remotely using one of the suggested [setup architectures][10].
+
+Datadog recommends you install the version listed in the [latest Agent version](#latest-agent-version). It contains all the implemented Oracle monitoring features and bug fixes.
```

Comment:
sorry another nitpick, I think my previous comment here was confusing - let's have this match the heading i.e. "Latest Agent version"